tests/qchem/of_tests/test_convert_openfermion.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for functions needed for converting objects obtained from external libraries to a
+PennyLane object.
+"""
+# pylint: disable=too-many-arguments,protected-access
+import sys
+
+import pytest
+
+import pennylane as qml
+from pennylane import fermi
+
+openfermion = pytest.importorskip("openfermion")
+
+
+# PennyLane operators were obtained from openfermion operators manually
+@pytest.mark.parametrize(
+    ("of_op", "pl_op"),
+    [
+        (
+            openfermion.FermionOperator("3^ 2", 0.5) + openfermion.FermionOperator("0 2^", 1.0),
+            fermi.FermiSentence(
+                {
+                    fermi.FermiWord({(0, 3): "+", (1, 2): "-"}): 0.5,
+                    fermi.FermiWord({(0, 0): "-", (1, 2): "+"}): 1.0,
+                }
+            ),
+        ),
+        (
+            openfermion.FermionOperator("1^ 2^ 0 3", 1.2)
+            + openfermion.FermionOperator("3^ 2^ 1 0", 4.5),
+            fermi.FermiSentence(
+                {
+                    fermi.FermiWord({(0, 1): "+", (1, 2): "+", (2, 0): "-", (3, 3): "-"}): 1.2,
+                    fermi.FermiWord({(0, 3): "+", (1, 2): "+", (2, 1): "-", (3, 0): "-"}): 4.5,
+                }
+            ),
+        ),
+        (
+            openfermion.FermionOperator("2^ 2", 1.5),
+            fermi.FermiSentence({fermi.FermiWord({(0, 2): "+", (1, 2): "-"}): 1.5}),
+        ),
+        (openfermion.FermionOperator("0^ 0"), fermi.FermiWord({(0, 0): "+", (1, 0): "-"})),
+        (
+            openfermion.FermionOperator("1^ 2", 1.5)
+            + openfermion.FermionOperator("2^ 1", 1.5)
+            + openfermion.FermionOperator("3^ 2", 0.8),
+            fermi.FermiSentence(
+                {
+                    fermi.FermiWord({(0, 1): "+", (1, 2): "-"}): 1.5,
+                    fermi.FermiWord({(0, 2): "+", (1, 1): "-"}): 1.5,
+                    fermi.FermiWord({(0, 3): "+", (1, 2): "-"}): 0.8,
+                }
+            ),
+        ),
+        (
+            openfermion.FermionOperator("3^ 4 5^ 6"),
+            fermi.FermiWord({(0, 3): "+", (1, 4): "-", (2, 5): "+", (3, 6): "-"}),
+        ),
+    ],
+)
+@pytest.mark.usefixtures("use_legacy_and_new_opmath")
```

Comment:
I am not 100% sure, but this shouldn't affect the conversion at all right? Probably could get rid of this fixture.

Reply:
This was removed, sorry this was missed earlier.